### PR TITLE
feat: added ratings component

### PR DESCRIPTION
**Pull request by kedark152**
### feat: added ratings component
1. Added Five Star Ratings with Ratings Result Text Besides it (Excellent, Very Good, Good, Bad & Very Bad).
2. This can be used for Product Review Ratings in E-commerce website. 

**Preview Link:** https://deploy-preview-7--forest-ui.netlify.app/docs-site/docs

### CODE REVIEW

File: stylesheets/components/ratings.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.demo-ratings{
+    display: inline-flex;
+    align-items: center;
+    padding: 0.2rem;
+    width: 18rem;
+    flex-wrap: nowrap;
+}
+
+.ratings .material-icons{
+
```
Comment (rahulcs754) [inline stylesheets/components/ratings.css]: remove space

File: stylesheets/components/ratings.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    flex-wrap: nowrap;
+}
+
+.ratings .material-icons{
+    
+    cursor: pointer;
+    color: var(--grey-lighter);
+    margin: 0.2rem;
+    font-size: 1.75rem;
+
```
Comment (rahulcs754) [inline stylesheets/components/ratings.css]: same as above

**Review by rahulcs754 — COMMENTED** [review]
Just small Changes

**Review by junaidshaikh-js — COMMENTED** [review]
Looks good to me!

File: stylesheets/components/ratings.css (hunk 1 above)
Comment (kedark152) [reply stylesheets/components/ratings.css]: updated!! please check & approve.

File: stylesheets/components/ratings.css (hunk 2 above)
Comment (kedark152) [reply stylesheets/components/ratings.css]: updated!! please check & approve.

**Review by junaidshaikh-js — APPROVED** [review]

**Review by rahulcs754 — APPROVED** [review]
now look all good

**Pull request merged**